core-primitives/sgx/crypto/src/rsa3072.rs

```diff
@@ -64,56 +65,88 @@ impl ShieldingCryptoEncrypt for Rsa3072PubKey {
 	}
 }
 
+impl ToPubkey for Rsa3072KeyPair {
+	type Error = Error;
+	type Pubkey = Rsa3072PubKey;
+
+	fn pubkey(&self) -> Result<Self::Pubkey> {
+		self.export_pubkey().map_err(|e| Error::Other(format!("{:?}", e).into()))
```

Comment:
We should be wary of continuing to use `Other` so often. This goes for myself included.

Reply:
Yes I know...